charts/trino/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "trino.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
why not use `networking.k8s.io/v1` available from 1.19-0? It's been available for quite a while.

Reply:
I will update this

Reply:
@yanyixing could you please update this version.